packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -89,8 +91,10 @@ export const SpacePlugin = ({
   const settings = new LocalStorageStore<SpaceSettingsProps>(SPACE_PLUGIN);
   const state = deepSignal<PluginState>({
     awaiting: undefined,
-    viewers: [],
-  }) as RevertDeepSignal<PluginState>;
+    viewersByObject: new Map(),
+    viewersByIdentity: new ComplexMap(PublicKey.hash),
+    // TODO(thure): is there a better way to cast this?
+  }) as unknown as PluginState;
```

Comment:
`RevertDeepSignal`

Reply:
Maybe let’s pair on this, I ended up needing to use a `Record` object literal for `viewersByObject` and couldn’t find a way around casting this as `unknown as PluginState`.